docs/pages/en-us/Architecture.md

```diff
@@ -5,78 +5,224 @@ title: Architecture
+                    "type": "text"
+                  }
+                }
+              },
+              {
+                "CAFAuditLong": {
+                  "match": "*_CALng",
+                  "mapping": {
+                    "type": "long"
+                  }
+                }
+              },
+              {
+                "CAFAuditInteger": {
+                  "match": "*_CAInt",
+                  "mapping": {
+                    "type": "integer"
+                  }
+                }
+              },
+              {
+                "CAFAuditShort": {
+                  "match": "*_CASrt",
+                  "mapping": {
+                    "type": "short"
+                  }
+                }
+              },
+              {
+                "CAFAuditDouble": {
+                  "match": "*_CADbl",
+                  "mapping": {
+                    "type": "double"
+                  }
+                }
+              },
+              {
+                "CAFAuditFloat": {
+                  "match": "*_CAFlt",
+                  "mapping": {
+                    "type": "float"
+                  }
+                }
+              },
+              {
+                "CAFAuditDate": {
+                  "match": "*_CADte",
+                  "mapping": {
+                    "type": "date"
+                  }
+                }
+              },
+              {
+                "CAFAuditBoolean": {
+                  "match": "*_CABln",
+                  "mapping": {
+                    "type": "boolean"
+                  }
+                }
+              }
+            ],
+            "properties": {
+              "applicationId": {
+                "type": "keyword"
+              },
+              "correlationId": {
+                "type": "keyword"
+              },
+              "eventCategoryId": {
+                "type": "keyword"
+              },
+              "eventOrder": {
+                "type": "integer"
+              },
+              "eventTime": {
+                "type": "date"
+              },
+              "eventTimeSource": {
+                "type": "keyword"
+              },
+              "eventTypeId": {
+                "type": "keyword"
+              },
+              "processId": {
+                "type": "keyword"
+              },
+              "threadId": {
+                "type": "keyword"
+              },
+              "userId": {
+                "type": "keyword"
+              }
+            }
+          }
+        }
+      }
+    }
+
+The above JSON, returned from Elasticsearch, shows us the fixed and custom field type mappings for a tenant's index. An audit event's information is stored in fixed fields and the event's parameters are mapped to their appropriate types based on their field name suffixes added by the Audit library.
+
+A tenant application's audit events sent from the client-side library are added that tenant's index.
```

Comment:
"An application's audit events for a tenant are sent from the client-side library to Elasticsearch and added to the index created for the tenant."

Or such like ...